.github/workflows/shared_deploy_cloudhub_1_0.yml

```
@@ -29,6 +29,11 @@ on:
         description: Main APIKit interface path
         type: string
         required: false
+      timeout-minutes:
```

Should it be `snake_case` to be consistent instead?
```suggestion
      timeout_minutes:
```
Or we can eliminate the word `minutes`, as it also is described below.
```suggestion
      timeout:
```

---

Sure @sanG-github - will rename it to `timeout_minutes`

---

Changed in https://github.com/nimblehq/mulesoft-actions/pull/20/commits/2be4c01af635fc34cee35e1642c4563c14f9c02e @sanG-github